WebApplication/Backend/Util/MyDbContext.cs

```diff
@@ -10,6 +12,9 @@ public class MyDbContext : DbContext
     {
```

Comment:
The class name should be more meaningful, like HospitalEditorDbContext

Reply:
This is the part of the web team, we will move our base to the backend. This was just a test for entities and mysql because we found out better solution. :)